src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini

```diff
@@ -15,7 +15,7 @@ COM_TJNOTIFICATIONS_FIELD_CLIENT_LABEL="Client"
 COM_TJNOTIFICATIONS_FIELD_KEY_LABEL="Key"
 COM_TJNOTIFICATIONS_FIELD_TITLE_LABEL="Title"
 COM_TJNOTIFICATIONS_FIELD_CLIENT_DESC="The client (extension) that is defining the notifications"
-COM_TJNOTIFICATIONS_FIELD_KEY_DESC="The unique key for the notification. This is unique within a client"
+COM_TJNOTIFICATIONS_FIELD_KEY_DESC="The key that is defining the notifications"
```

Comment:
change as

`The unique key that is defining the notifications. This is unique within a client.`